script/import_scripts/socialcast/test/test_data.rb

```diff
@@ -8166,8 +8166,8 @@
                         "back_in_office_on": null,
                         "company_login": null
                     },
-                    "text": "After speaking with a number of our younger employees I think tutition reimbursement is really key to retaining them. Many of them are looking to earn advanced degrees. If we can setup a plan that accomplishes this effectively we can really grow our staff\'s ability and knowledge.",
-                    "html_body": "<div class=\"markdown-body\"><p>After speaking with a number of our younger employees I think tutition reimbursement is really key to retaining them. Many of them are looking to earn advanced degrees. If we can setup a plan that accomplishes this effectively we can really grow our staff\'s ability and knowledge.</p></div>",
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617142496
Are the misspellings important?